Sources/Moho/Name.swift

```diff
@@ -108,3 +113,9 @@ public struct QualifiedName: Equatable, Hashable, CustomStringConvertible {
     return self.description
   }
 }
+
+extension QualifiedName: ExpressibleByStringLiteral {
```

Comment:
Please don’t do this.

Reply:
Yeah, this is _only_ for testing and I'm gonna remove it.